README.md

```diff
@@ -1,16 +1,54 @@
 # Release Integration Tests
 
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## 1- Fbc happy path e2e-test ##
+
+ This section provides instructions on running the FBC e2e test with the provided script
+
+   ## Test Steps and Expected Result
+
+   1. **Pre-Running Test:** The following setup actions need to be completed before running the test script.
+      
+      The test is running against RHTAP-Stage, so a user who wants to monitor the test running through the UI or the CLI. 
+      will need access to the cluster and the workspaces *release-dev-team* and *release-managed-team*. 
```

Comment:
Isn't it `managed-release-team-tenant` and `dev-release-team-tenant`?

Reply:
> Isn't it `managed-release-team-tenant` and `dev-release-team-tenant`?

those are namespaces, this why I wrote `workspaces`

Reply:
Hum oki. So I guess this would need elaborate or link how to properly login into these workspaces. I ran your script locally just defining the namespaces, but not workspaces. Well, maybe I am the only one that did not understand this.

Reply:
This test is targeted to run against RHTAP-Stage for now ,  we can add a requirement to be able to run on local clusters but not sure if we want to do it in term of trade-off, for this we want to have a team discussion .
another option is that you ask to be joined to the dedicated workspaces in RHTAP-Stage and you can run the test too .

Reply:
Me and @kasemAlem had a quick call so I could explain my POV. Closing this comment.

Reply:
this is still wrong

workspace is **dev-release-team**
namespace is **dev-release-team-tenant**

Reply:
yes, 
my bad 
thanks